src/components/QueryExecutionStatus/QueryExecutionStatus.tsx

```diff
@@ -1,29 +1,46 @@
+import type {ReactNode} from 'react';
 import cn from 'bem-cn-lite';
 
+import {Icon as UiKitIcon} from '@gravity-ui/uikit';
+
 import Icon from '../Icon/Icon';
 
+import questionIcon from '../../assets/icons/question.svg';
+
 import './QueryExecutionStatus.scss';
 
 const b = cn('kv-query-execution-status');
 
 interface QueryExecutionStatusProps {
     className?: string;
-    hasError?: boolean;
+    error?: Record<string, any>;
```

Comment:
I think It is better to introduce some interface `IQueryResponseError` and at least specify  properties that could be present. It makes it much easier to understand